### ISSUE TITLE
Object requires a newer version of iOS

### ISSUE THREAD

**Issue by ermincelikovic** [google/model-viewer#4772]
### Description

Hi Folks,
first of all, thank you for this wonderful library!

However, I'm not sure what am I doing wrong since my AR on iOS does not seem to work - it returns the message:
`Object requires a newer version of iOS.`

I assume that something might be wrong with my model:
https://cdn.glitch.global/40b96037-9b1b-4d40-8f86-6c16abe963fc/model.glb?v=1715287619810

Once I try to run it in AR, I can find following warning in my console:
`THREE.USDZExporter: USDZ does not support double sided materials`
When I set double sided material to false, the AR still shows "Object requires a newer version of iOS".

To be clear, I’m on the latest version of iOS and i tried several others too.

Thanks in advance for any help!

#### Live Demo

https://smooth-abalone-molybdenum.glitch.me/

### Version

- model-viewer: v3.5.0

### Browser Affected

- [ ] Chrome, version: xx.x.xxxx.xx
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [x] Safari

### OS

- [ ] Android
- [x] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR

- [ ] WebXR
- [ ] SceneViewer
- [x] QuickLook

**Comment by elalish** [comment]
Considering it says you need a newer version of iOS, could you at least include your iOS version? :)

@AdaRoseCannon @mrdoob Maybe you two can work out some kind of testing regimen so that Apple's Quick Look isn't constantly making breaking changes to its USDZ reader (this file is produced by Three's USDZExporter)?

**Comment by AdaRoseCannon** [comment]
Just to clarify, what version of THREE.js does this USDZExporter come from?

**Comment by elalish** [comment]
r163 - we're generally up to date at whatever the time of our release was.

**Comment by mrdoob** [comment]
From my side I have no other option than reverse engineer iOS viewer VR disabling things until it works again. It would be greatly appreciated if iOS could offer a mechanism to know what the issue actually is.

**Comment by AdaRoseCannon** [comment]
Can you link the USDZ model generated so we can take a look?

**Comment by elalish** [comment]
@AdaRoseCannon do you have a tool to extract it from a QuickLook session? MV just generates it in memory and passes the blob across, so there isn't super easy file access without hacking MV a bit. You might consider setting up a Glitch with Three.js for converting glTF to USDZ so you can easily repro problems like this (they are very common with our users).

**Comment by AdaRoseCannon** [comment]
I’ll do that, if @ermincelikovic has the original gltf then that would also be fine and I will make a glitch to test it with.

**Comment by elalish** [comment]
I believe the original GLB is the first link in the description above.

**Comment by AdaRoseCannon** [comment]
Totally missed it, my bad!

**Comment by mrdoob** [comment]
> @AdaRoseCannon do you have a tool to extract it from a QuickLook session? MV just generates it in memory and passes the blob across, so there isn't super easy file access without hacking MV a bit. You might consider setting up a Glitch with Three.js for converting glTF to USDZ so you can easily repro problems like this (they are very common with our users).

Or you can load the glb in https://threejs.org/editor/ and export to usdz from there too.

**Comment by AdaRoseCannon** [comment]
That is a really useful tip and will save a lot of time.

**Comment by elalish** [comment]
Yes, I think we've been told to file Apple issues through their feedback assistant, though I personally find that kind of one-way communication irritating, so I'd also recommend filing it on their WebKit bug tracker where you might at least get an update.
